server/simple_server.go

```diff
@@ -112,6 +112,8 @@ func (s *SimpleServer) Start() error {
 	srv := http.Server{
 		Handler:        RegisterAccessLogger(s.cfg, s),
 		MaxHeaderBytes: maxHeaderBytes,
+		ReadTimeout:    readTimeout,
+		WriteTimeout:   writeTimeout,
```

Comment:
The values in this `http.Server` struct are duplicated here and server/rpc_server.go. It's not a big deal, but if they're always going to be synchronized, it might be worth extracting to some other place.